operators/src/source/gdal_source/mod.rs

```diff
@@ -1616,4 +1789,199 @@ mod tests {
+            read_start_x: 0,
+            read_start_y: 0,
+        };
+
+        assert_eq!(rw, exp);
+    }
+
+    #[test]
+    fn read_up_side_down_raster() {
+        let output_shape: GridShape2D = [8, 8].into();
+        let output_bounds =
+            SpatialPartition2D::new_unchecked((-180., 90.).into(), (180., -90.).into());
+
+        let up_side_down_params = GdalDatasetParameters {
+            file_path: test_data!(
+                "raster/modis_ndvi/flipped_axis_y/MOD13A2_M_NDVI_2014-01-01_flipped_y.tiff"
+            )
+            .into(),
+            rasterband_channel: 1,
+            geo_transform: GdalDatasetGeoTransform {
+                origin_coordinate: (-180., -90.).into(),
+                x_pixel_size: 0.1,
+                y_pixel_size: 0.1,
+            },
+            width: 3600,
+            height: 1800,
+            file_not_found_handling: FileNotFoundHandling::NoData,
+            no_data_value: Some(0.),
+            properties_mapping: Some(vec![
+                GdalMetadataMapping {
+                    source_key: RasterPropertiesKey {
+                        domain: None,
+                        key: "AREA_OR_POINT".to_string(),
+                    },
+                    target_type: RasterPropertiesEntryType::String,
+                    target_key: RasterPropertiesKey {
+                        domain: None,
+                        key: "AREA_OR_POINT".to_string(),
+                    },
+                },
+                GdalMetadataMapping {
+                    source_key: RasterPropertiesKey {
+                        domain: Some("IMAGE_STRUCTURE".to_string()),
+                        key: "COMPRESSION".to_string(),
+                    },
+                    target_type: RasterPropertiesEntryType::String,
+                    target_key: RasterPropertiesKey {
+                        domain: Some("IMAGE_STRUCTURE_INFO".to_string()),
+                        key: "COMPRESSION".to_string(),
+                    },
+                },
+            ]),
+            gdal_open_options: None,
+            gdal_config_options: None,
+            allow_alphaband_as_mask: true,
+        };
+
+        let tile_information =
+            TileInformation::with_partition_and_shape(output_bounds, output_shape);
+
+        let RasterTile2D {
+            global_geo_transform: _,
+            grid_array: grid,
+            tile_position: _,
+            time: _,
+            properties: _,
+        } = GdalRasterLoader::load_tile_data::<u8>(
+            &up_side_down_params,
+            tile_information,
+            TimeInterval::default(),
+        )
+        .unwrap();
+
+        assert!(!grid.is_empty());
+
+        let grid = grid.into_materialized_masked_grid();
+
+        assert_eq!(grid.inner_grid.data.len(), 64);
+        assert_eq!(
+            grid.inner_grid.data,
+            &[
+                255, 255, 255, 255, 255, 255, 255, 255, 255, 75, 37, 255, 44, 34, 39, 32, 255, 86,
```

Comment:
How did you verify the data? Maybe you could add a dataset definition so that we could see in the UI that it's correct?

Reply:
it is a flipped version of the dataset we use for all the other tests. it reads the flipped data and produces the same output as the non-flipped data.

Reply:
[add ndvi flipped y-axis def](https://github.com/geo-engine/geoengine/pull/602/commits/1ad08331349d8e7505cabfd1f9f9d6a4af1ff4f7)